src/translation.cpp

```diff
@@ -0,0 +1,760 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "translation.h"
+
+// Headers
+#include <fstream>
+#include <iomanip>
+#include <lcf/data.h>
+#include <lcf/rpg/terms.h>
+#include <lcf/rpg/map.h>
+#include "lcf/rpg/mapinfo.h"
+
+#include "cache.h"
+#include "main_data.h"
+#include "game_actors.h"
+#include "game_map.h"
+#include "player.h"
+#include "output.h"
+#include "utils.h"
+
+
+// Name of the translate directory
+#define TRDIR_NAME "languages"
+
+// Name of expected files
+#define TRFILE_RPG_RT_LDB    "rpg_rt.ldb.po"
+#define TRFILE_RPG_RT_COMMON "rpg_rt.ldb.common.po"
+#define TRFILE_RPG_RT_BATTLE "rpg_rt.ldb.battle.po"
+#define TRFILE_RPG_RT_LMT    "rpg_rt.lmt.po"
+#define TRFILE_META_INI      "META.INI"
+
+// Message box commands to remove a message box or add one in place.
+#define TRCUST_REMOVEMSG        "__EASY_RPG_CMD:REMOVE_MSGBOX__"
+#define TRCUST_ADDMSG           "__EASY_RPG_CMD:ADD_MSGBOX__"
+
+
+std::string Tr::TranslationDir() {
```

Comment:
Is a getter, call it ``GetTranslationDir``

Reply:
Fixed